test/conv_transpose2d_test.js

```diff
@@ -0,0 +1,585 @@
+'use strict';
+
+import {convTranspose2d} from '../src/conv_transpose2d.js';
+import {clamp} from '../src/clamp.js';
+import {leakyRelu} from '../src/leaky_relu.js';
+import {relu} from '../src/relu.js';
+import {sigmoid} from '../src/sigmoid.js';
+import {Tensor} from '../src/lib/tensor.js';
+
+import * as utils from './utils.js';
+
+describe('test convTranspose2d', function() {
```

Comment:
What are `describe` and `it`? It's a different way of listing the test data than most *previous* files, which were more data driven. (not opposing it, but curious)